tests/test_sockets.py

```diff
@@ -53,7 +53,9 @@ def __enter__(self):
     # NOTE empty filename support is a hack to support
     # the current test_enet
     if self.filename:
-      run_process([CLANG_CC, test_file(self.filename), '-o', 'server', '-DSOCKK=%d' % self.target_port] + clang_native.get_clang_native_args() + self.args, env=clang_native.get_clang_native_env())
+      cmd = [CLANG_CC, test_file(self.filename), '-o', 'server', '-DSOCKK=%d' % self.target_port] + clang_native.get_clang_native_args() + self.args
+      print(cmd)
```

Comment:
is it intentional to leave this print in place? (fine with me if so, just checking)

Reply:
Yup, we want to see the command name.  But I just remembered we have self.run_process for this.. so I'll switch to that.

Reply:
Actually this code is not withing a test suite.. so I can't get to self.run_process from here.